docs/_layouts/default.html

```diff
@@ -30,7 +30,7 @@
           </a>
         </div>
       </div>
-      <a href="{{ site.baseurl }}/{{ lang }}/">
+      <a href="{{ site.baseurl }}/{{ lang }}/" style="color: #ffffff; text-decoration: none;">
```

Comment:
Please change the style in css/screen.scss instead of inline in the HTML template.